ppoprf/src/ppoprf.rs

```diff
@@ -100,6 +300,17 @@ impl ProofDLEQ {
     RistrettoScalar::from_bytes_mod_order_wide(&out)
   }
 
+  // I2OSP(x, xLen): Converts a non-negative integer x into a byte
+  // array of specified length xLen as described in [RFC8017]. Note
+  // that this function returns a byte array in big-endian byte
+  // order.
```

Comment:
I think this depends on the caller invoking `.to_be_bytes()` to get correct big-endian output? Maybe it would be better to take `x: usize` by value and return an array instead?

Or, since the algorithm only uses x_len = 2, specializing to that as `i2osp2` avoids the endianness issue.